src/neptune_prophet/impl/__init__.py

```diff
@@ -56,15 +54,17 @@
     from neptune.types import File, FloatSeries
 
 from prophet import Prophet
-
 from prophet.plot import (
     add_changepoints_to_plot,
     plot_components_plotly,
     plot_plotly,
 )
-
 from prophet.serialize import model_to_json
 
+from neptune_prophet._version import get_versions
```

Comment:
There is no use of `__version__` in Prophet (I think so) so let's leave this in main `__init__.py`